salt/master.py

```diff
@@ -349,8 +349,8 @@ class FileserverUpdate(salt.utils.process.SignalHandlingMultiprocessingProcess):
     '''
     A process from which to update any dynamic fileserver backends
     '''
-    def __init__(self, opts, log_queue=None):
-        super(FileserverUpdate, self).__init__(log_queue=log_queue)
+    def __init__(self, opts, **kwargs):
```

Comment:
I do not like the idea of using kwargs for something like this, especially without documenting what can be passed as kwargs.

Reply:
All classes inherited from salt/utils/process.py:MultiprocessingProcess must use **kwargs after commit 90edc697c2a2dd21fbc5f7a139139e227fd025eb

Reply:
Perfect, thank you for the explanation.  Now it makes sense on why you are using `**kwargs`

Reply:
Sure! I'll add the explanation in the description. Sorry for the misunderstanding.

Reply:
Awesome!  Thank you very much for this addition!

Daniel